0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -22,81 +22,42 @@ SPSP messages MUST be exchanged over HTTPS.
 
 ## Overview
 
 ### Relation to Other Protocols
 
-SPSP is used for exchanging payment information before an ILP payment is initiated. It uses the receiver generated [Interledger Payment Requests](../0011-interledger-payment-request) over HTTP.
+SPSP is used for exchanging payment information before an ILP payment is initiated. The sender and recipient use the [Pre-Shared Key (PSK)](../0016-pre-shared-key/0016-pre-shared-key.md) transport protocol to generate the condition and fulfillment. The recipient generates the shared secret to be used in PSK and communicates it to the sender over HTTPS.
 
 ### Model of Operation
 
-#### Fixed Destination Amount
-
-SPSP may be used by the sender to send a payment such that the recipient receives a specific amount of their chosen currency or asset type. The model of operation is illustrated with the following example:
-
 We assume that the sender knows the receiver endpoint (see [Appendix A: (Optional) Webfinger Discovery](#appendix-a-optional-webfinger-discovery)).
 
 1. The sender's SPSP client queries the receiver endpoint.
-2. The receiver endpoint responds with the receiver info, including the receiver's currency code.
-3. The sender chooses an amount of the receiver's asset to deliver.
-4. The sender's SPSP client submits the payment information, including the destination amount, to the receiver endpoint.
-5. The receiver endpoint responds with an [Interledger Payment Request](../0011-interledger-payment-request), which includes the execution condition.
-6. The sender's SPSP client uses its ILP module to get a quote in their currency or asset type for the ILP transfer.
-7. The sender accepts the quote.
-8. The sender's SPSP client uses its ILP module to initiate the ILP payment.
-9. The receiver's ILP module registers the incoming transfer held pending the fulfillment of the Crypto Condition. It validates that the transfer matches the packet and regenerates the condition fulfillment using the method recommended in the [Interledger Payment Request](../0011-interledger-payment-request) spec. The ILP module submits the fulfillment to execute the transfer and claim the funds.
+2. The receiver endpoint responds with the receiver info, including the recipient's ILP address and the shared secret to be used in PSK.
+3. The sender constructs an ILP payment using the recipient's ILP address.
+4. The sender uses PSK to generate the payment condition and format additional data to be sent with the payment.
+5. The sender uses their ILP module to initiate the ILP payment.
+6. The receiver's ILP module registers the incoming transfer held pending the fulfillment of the condition.
+7. The receiver's PSK module parses and validates the ILP packet and the incoming transfer.
+8. The receiver may submit the incoming payment to an external system for review to ensure that the funds are wanted.
```

Comment:
I think those are worth including in here because they're things you would need to handle when implementing an SPSP server. Just because the current implementation of the PSK client does these things and provides a callback that you'd use to insert the review step, it could also be implemented as separate steps that you need to call from the SPSP server.